package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-multi-dotnet",
-  "version": "0.9.12",
+  "version": "0.9.11",
```

Comment:
Why lower the number?

Reply:
This is a mistake, fixed to 0.9.13.
Can write another version number if you prefer